.github/workflows/node.js.yml

```
@@ -17,10 +17,10 @@ jobs:
       with:
         cache: npm
```

Hi! If you change from npm to yarn use it also in the cache system 
```suggestion
        cache: yarn
```